packages/react/src/index.tsx

```diff
@@ -1,3 +1,5 @@
+import './styles.css';
```

Comment:
This is really nice – building (via PostCSS) styles as part of a single package & process.

Reply:
Somehow the SCSS stuff just worked?? I need to check the docs to be sure...

Reply:
Good question @dbanksdesign!  No, `import '@aws-amplify/ui-react/styles.css'` is still required.

What `import` does here is allow the bundler (`tsup`, `microbundle`, whatever) automatically output both JS & CSS with a single process. (Compared to having to watch CSS **and** JS separately)

This is consistent with how, say, an application is bundled with webpack: even though your app has JS, CSS, SVGs, PNGs, whatever, the bundler identifies all dependencies and outputs them based on your configuration.